README.md

```diff
@@ -60,6 +60,36 @@ client.trackMetric("custom metric", 3);
 client.trackTrace("trace message");
 ```
 
+### Telemetry Processor
+
+```javascript
+public addTelemetryProcessor(telemetryProcessor: (envelope: ContractsModule.Contracts.Envelope) => boolean)
+```
+
+Adds telemetry processor to the collection. Telemetry processors will be called one by one before telemetry item is pushed for sending and in the order they were added. 

```

Comment:
"Adds a telementry..." ... "...before the telemetry..."
I might also move the bit about "in the order they were added" to just after "one by one" so it reads "...one by one, in the order they were added, before the telemetry item is pushed...".